models/mit_ipea_project/vw_ticketing.sql

```diff
@@ -0,0 +1,40 @@
+-- ticketing view, ticketing data cleaned for:
+-- - card types that do not unique identify users (these include cash transactions)
+-- - pre-identified card ids with extreme values
+
+WITH raw_ticketing AS (SELECT data                                                               AS as_at,
+                              EXTRACT(time from datetime)                                        AS origin_time,
+                              id_cartao                                                          AS card_id,
+                              tipo_cartao                                                        AS card_type,
+                              ROW_NUMBER() OVER (PARTITION BY id_cartao, data ORDER BY datetime) AS daily_trip_id,
+                              id_empresa                                                         AS company_id,
+                              id_veiculo                                                         AS vehicle_id
+
+                       FROM `rj-smtr.br_rj_riodejaneiro_onibus_tg.transacao`
```

Comment:
Change to ref

Reply:
It looks like `master` branch doesn't have this model? I think branch `tg` contains this model? Am I still able to ref this model?

Reply:
You're right